src/mem/mem_ctrl.hh

```diff
@@ -251,15 +252,14 @@ class MemCtrl : public QoS::MemCtrl
 
       protected:
 
-        Tick recvAtomic(PacketPtr pkt) override;
-        Tick recvAtomicBackdoor(
-                PacketPtr pkt, MemBackdoorPtr &backdoor) override;
+        Tick recvAtomic(PacketPtr pkt);
+        Tick recvAtomicBackdoor(PacketPtr pkt, MemBackdoorPtr &backdoor);
 
-        void recvFunctional(PacketPtr pkt) override;
+        void recvFunctional(PacketPtr pkt);
```

Comment:
Why did you remove the override keywords? These are quite important. https://en.cppreference.com/w/cpp/language/override